client/rest-high-level/src/main/java/org/elasticsearch/client/transform/transforms/SettingsConfig.java

```diff
@@ -176,8 +202,24 @@ public Builder setDatesAsEpochMillis(Boolean datesAsEpochMillis) {
             return this;
         }
 
+        /**
+         * Whether to write the output of a date aggregation as millis since epoch or as formatted string (ISO format).
+         *
+         * Transforms created before 7.11 write dates as epoch_millis. The new default is ISO string.
+         * You can use this setter to configure the old style writing as epoch millis.
+         *
+         * An explicit `null` resets to default.
+         *
+         * @param interimResults true if dates should be written as epoch_millis.
+         * @return the {@link Builder} with datesAsEpochMilli set.
+         */
```

Comment:
This comment has been pasted from above and needs updating.

Reply:
Done.